docs/data/sdks/typescript-browser/index.md

```diff
@@ -557,15 +1170,24 @@ You need to download `plugin-web-attribution-browser` package and add the `webAt
     yarn add @amplitude/plugin-web-attribution-browser
     ```
 
-```ts
-import * as client from '@amplitude/analytics-browser';
-import { add, init } from '@amplitude/analytics-browser';
-import { webAttributionPlugin } from '@amplitude/plugin-web-attribution-browser';
+Add plugin to Ampltude instance.
```

Comment:
typo 
```suggestion
Add plugin to Amplitude instance.
```

Reply:
Miss `i` in Amplitude. 
Should be Amplitude instead of Ampltude.